docs/autogenerate_OAI_models.md

```diff
@@ -0,0 +1,22 @@
+## Motivation
+You will only need to use the script to build models when any of the GA4GH API's openapi specs are updated. The models are being used to return the result of any API call in the correct format, in the correct structs
+
+
+## Usage
```

Comment:
I don't think there are any prerequisites except making sure that script.sh has the necessary permissions, and I have changed that

Reply:
@pavelnikonorov, can you please look into this, and tell me if there are any other prerequisites?

Reply:
@aaravm I agree that OpenAPI can be a required dependency for a user to install beforehand, aside from this, Java is the only other as I see from scripts

Reply:
Sure, there may only be these two dependencies. But neither Java/JDK nor the OpenAPI generator is listed as a dependency. If someone follows these instructions, it is quite likely that they will fail because of missing deps.

And for the reasons mentioned elsewhere, I don't think it is a good idea to install dependencies in a script.

Reply:
@aaravm as I understood we decided to remove the installation of Java and OpenAPI generator from the scripts. I see documentation for Java installation. But there is no documentation on OpenAPI gen (cli or jar) and it's still installed by the script.